tests/Language/PureScript/Ide/CompletionSpec.hs

```diff
@@ -6,9 +6,12 @@ import Protolude
 
 import Language.PureScript as P
 import Language.PureScript.Ide.Completion
-import Language.PureScript.Ide.Test
+import Language.PureScript.Ide.Test as Test
+import           Language.PureScript.Ide.Command as Command
```

Comment:
The import formatting is inconsistent here, I don't mind leaving space for `qualified`, but let's apply that formatting on every import in this file then.